api/docker-compose.yml

```
@@ -1,27 +1,169 @@
+  #      - web-network
+  #    depends_on:
+  #      - api
 
   redis:
-    image: redis:alpine
-    container_name: redis_dev
+    container_name: redis-container
+    image: redis:latest
     volumes:
       - redis-volume:/data
+    command: [ "redis-server", "--appendonly", "yes" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)